src/Concerns/WithInertiaController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Orchestra\Testbench\Concerns;
+
+use Illuminate\Support\Facades\View;
+use Inertia\ServiceProvider;
+
+trait WithInertiaController
+{
+    public function loadInertia(
+        string $viewPath = 'resources/views',
+        array $pathToPages = ['resources/js/Pages', 'resources/js/pages', 'resources/ts/Pages', 'resources/ts/pages', 'resources/app/Pages', 'resources/app/pages', 'resources/views/pages',],
+        array $fileFinderExtensions = ['vue', 'js', 'jsx', 'ts', 'tsx', 'html', 'php']
+    ) {
+        View::addLocation($viewPath);
+        $this->app->bind('inertia.testing.view-finder', function ($app) use ($pathToPages, $fileFinderExtensions) {
+            $viewFinder = new \Illuminate\View\FileViewFinder(
+                $app['files'],
+                array_merge($app['config']->get('inertia.testing.page_paths'), $pathToPages),
+                array_merge($app['config']->get('inertia.testing.page_extensions'), $fileFinderExtensions)
+            );
+            $viewFinder->addNamespace('WD', 'UI/resources/app/Pages');
```

Comment:
This shouldn't hardcoded.

Reply:
You are right sorry this slipped through. I made this as an optional param.

Reply:
@crynobone Will I also need to add an example or is it fine like this?